packages/core/src/external/quest/fhir/condition.ts

```diff
@@ -0,0 +1,63 @@
+import { uuidv7 } from "@metriport/shared/util/uuid-v7";
+import { Condition, Patient } from "@medplum/fhirtypes";
+import { ResponseDetail } from "../schema/response";
+import { ICD_10_URL, ICD_9_URL } from "@metriport/shared/medical";
+import { getPatientReference } from "./patient";
+import { getQuestDataSourceExtension } from "./shared";
+
+export function getConditions(
+  detail: ResponseDetail,
+  { patient }: { patient: Patient }
+): Condition[] {
+  const conditions = [
+    getCondition(patient, detail.diagnosisCode1),
+    getCondition(patient, detail.diagnosisCode2),
+    getCondition(patient, detail.diagnosisCode3),
+    getCondition(patient, detail.diagnosisCode4),
+    getCondition(patient, detail.diagnosisCode5),
+    getCondition(patient, detail.diagnosisCode6),
+    getCondition(patient, detail.diagnosisCode7),
+    getCondition(patient, detail.diagnosisCode8),
+    getCondition(patient, detail.diagnosisCode9),
+    getCondition(patient, detail.diagnosisCode10),
+  ].filter(Boolean) as Condition[];
+
+  return conditions;
```

Comment:
do _([
    getCondition(patient, detail.diagnosisCode1),
    getCondition(patient, detail.diagnosisCode2),
    getCondition(patient, detail.diagnosisCode3),
    getCondition(patient, detail.diagnosisCode4),
    getCondition(patient, detail.diagnosisCode5),
    getCondition(patient, detail.diagnosisCode6),
    getCondition(patient, detail.diagnosisCode7),
    getCondition(patient, detail.diagnosisCode8),
    getCondition(patient, detail.diagnosisCode9),
    getCondition(patient, detail.diagnosisCode10),
  ]).compact()

to avoid using the type assertion